modules/nf-core/jvarkit/vcfpolyx/environment.yml

```diff
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "jvarkit_vcfpolyx"
+channels:
+  - bioconda
```

Comment:
just bioconda?

Reply:
@maxulysse isn't there a problem with conda that is now commercial ?  and I there is currently a problem with environment.yml...
```
: 'conda-forge' is │ │                   │                            │ not one of ['bioconda']. 
```